plugins/browser-plugin-error-tracking/src/index.ts

```diff
@@ -147,15 +147,35 @@ function sendError(
     context = context.concat(contextAdder(errorEvent));
   }
 
-  trackError(
-    {
-      message: errorEvent.message,
-      filename: errorEvent.filename,
-      lineno: errorEvent.lineno,
-      colno: errorEvent.colno,
-      error: errorEvent.error,
-      context,
-    },
-    trackers
-  );
+  if ('message' in errorEvent) {
+    trackError(
+      {
+        message: errorEvent.message,
+        filename: errorEvent.filename,
+        lineno: errorEvent.lineno,
+        colno: errorEvent.colno,
+        error: errorEvent.error,
+        context,
+      },
+      trackers
+    );
+  } else if (errorEvent.target && 'tagName' in errorEvent.target) {
+    const element: any = errorEvent.target;
+    trackError(
```

Comment:
Might have missed this, but can we cover this branch (and perhaps also the one below) with a unit test as well?

Reply:
Just realized that the `sendError` function is not exported. Leaving the comment here if you can trigger the event listener in an easy way and test the whole handling, otherwise feel free to skip this.